server/src/messageHandler.mjs

```diff
@@ -117,43 +104,31 @@ async function handleMessage(message, userId, ws) {
     logger.info(`Sent "invalid params" message: ${responseMessage}`);
   }
 
-  // Fire pre trigger if there is one for this method
-  let shouldContinue;
-  if ( oMsg.method in triggers ) {
-    if ( 'pre' in triggers[oMsg.method] ) {
+  // Fire pre trigger if there is one for this method  
+  if ( oMsg.method in methodTriggers ) {
+    if ( 'pre' in methodTriggers[oMsg.method] ) {
       try {
-        const ctx = {
+        const ctx = {          
+          logger: logger,
           setTimeout: setTimeout,
           setInterval: setInterval,
           sendEvent: function(onMethod, result, msg) {
-            const id = events.getRegisteredEventListener(onMethod);
-            if ( ! id ) { return; }
-            emit(id, result, msg, ws);
+            function fSuccess() {}
+            function fErr() {}
```

Comment:
These empty implementations for fSuccess, fErr and fFatalErr are okay if you're sure that some kind of logging is happening inside the sendEvent() method where these methods are called.

Reply:
added logger statements in fSuccess, fErr and fFatalErr